pkg/api/admin/convertertointernal.go

```diff
@@ -90,6 +90,10 @@ func mergePropertiesAdmin(oc *OpenShiftManagedCluster, cs *api.OpenShiftManagedC
 	if oc.Properties.FQDN != nil {
 		cs.Properties.FQDN = *oc.Properties.FQDN
 	}
+	if oc.Properties.PrivateAPIServer != nil {
```

Comment:
I think we should drop this. This would allow admin to set cluster to the public, and potentially break the cluster.
This should be `read-only`

Reply:
@mjudeikis  Makes sense. I removed this from conversion to internal.

PTAL. Also note `TestRoundTrip`: had to add an exception.